SmartDeviceLink/SDLTouchManager.m

```diff
@@ -184,7 +185,8 @@ - (void)sdl_onTouchEvent:(SDLRPCNotificationNotification *)notification {
         return;
     }
 
-    SDLOnTouchEvent* onTouchEvent = (SDLOnTouchEvent*)notification.notification;
+    SDLOnTouchEvent *onTouchEvent = (SDLOnTouchEvent *)notification.notification;
+    onTouchEvent = [self sdl_applyScaleToEventCoordinates:onTouchEvent.copy];
```

Comment:
I don't think `copy` is working as expected here which is why the `TouchManager` test cases are breaking when I run all the tests. Maybe `copyWithZone:` has a bug?